x/fees/genesis.go

```diff
@@ -0,0 +1,47 @@
+package fees
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/tharsis/evmos/v3/x/fees/keeper"
+	"github.com/tharsis/evmos/v3/x/fees/types"
+)
+
+// InitGenesis import module genesis
+func InitGenesis(
+	ctx sdk.Context,
+	k keeper.Keeper,
+	accountKeeper authkeeper.AccountKeeper,
+	data types.GenesisState,
+) {
+	k.SetParams(ctx, data.Params)
+
+	// Ensure fees module account is set on genesis
+	if acc := accountKeeper.GetModuleAccount(ctx, types.ModuleName); acc == nil {
+		panic("the fees module account has not been set")
+	}
+
+	for _, fee := range data.DevFeeInfos {
+		var withdrawal *sdk.AccAddress
+		contract := common.HexToAddress(fee.ContractAddress)
+		deployer, _ := sdk.AccAddressFromBech32(fee.DeployerAddress)
+		_withdrawal, err := sdk.AccAddressFromBech32(fee.DeployerAddress)
+		if err == nil {
+			withdrawal = &_withdrawal
+		}
```

Comment:
```suggestion
		contract := common.HexToAddress(fee.ContractAddress)
		deployer, _ := sdk.AccAddressFromBech32(fee.DeployerAddress)
		withdrawal := deployer
		
		if fee.WithdrawAddress != "" {
    		withdrawal, _ = sdk.AccAddressFromBech32(fee.WithdrawAddress)
		}
```

Reply:
I fixed the redundant pointer in https://github.com/tharsis/evmos/pull/436/commits/ea82e4528d1b56ac9bec16b391b02b26a0df86e2. But without `withdrawal := deployer`. If `withdrawal` is nil, it is not stored in the keeper (to avoid the state bloat we talked about).